adoptable_modules.html.erb

```diff
@@ -0,0 +1,50 @@
+<html>
+<title>Puppetlabs modules up for adoption</title>
+<head>
```

Comment:
please add the GA tracker like in https://github.com/puppetlabs/community_management/blob/a537bf8c14c011537687c795678f00577a971798/index.html#L9-L20

Reply:
looking through the other reports, we need to add it to a couple of pages.